shopify/packages/shopify/shopify_products_sold.js

```diff
@@ -0,0 +1,132 @@
+// jshint esversion: 9
+
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+const axios = require('axios');
+
+async function getRequest(url) {
+
+	return await axios.get(url)
+		.then((response) => { return response.data; })
+		.catch((error) => { return error.response.data; });
+}
+
+function isSecretMissing(secrets) {
+
+	var ret = '';
+
+	if (!secrets.shopifyKey) {
+		ret += 'Shopify API key not found!\n';
+	} if (!secrets.shopifyPassword) {
+		ret += 'Shopify password not found!\n';
+	} if (!secrets.shopifyHostname) {
+		ret += 'Shopify hostname not found!\n';
+	}
+	return ret;
+}
+
+function parseOrdersForItemNames(orders) {
+
+	var itemsCount = [];
+
+	for (var i = 0; i < orders.length; i++) {
+		for (var j = 0; j < orders[i].line_items.length; j++) {
+			if (!itemsCount[orders[i].line_items[j].name]) {
+				itemsCount[orders[i].line_items[j].name] = 1;
+			} else {
+				itemsCount[orders[i].line_items[j].name] += 1;
+			}
+		}
+	}
+	return itemsCount;
+}
+
+function filterItems(search_terms, itemsCount) {
+
+	var filteredItems = [];
+	var keys = Object.keys(itemsCount);
+
+	for (var i = 0; i < keys.length; i++) {
+		for (var j = 0; j < search_terms.length; j++) {
+			if (keys[i].toLowerCase().includes(search_terms[j].toLowerCase())) {
+				filteredItems.push(keys[i]);
+			}
+		}
+	}
+	return [...new Set(filteredItems)];
+}
+
+function formatReturnText(filteredItems, itemsCount) {
+
+	var itemsArray = [];

```

Comment:
```suggestion
	const itemsArray = [];

```

Reply:
I was not aware const could work in cases like this, nice find.